appium-uiautomator2-server/build.gradle

```diff
@@ -37,3 +37,17 @@ dependencies {
     androidTestCompile 'com.android.support.test.espresso:espresso-core:2.0'
 
 }
+
+task installAUT(type: Exec) {
```

Comment:
I think we'd want a new test flavor. See [testFlavor1](https://github.com/googlesamples/android-testing-templates/tree/master/AndroidTestingBlueprint/app/src) from the android blueprint.

If the source set works though then that might be good enough for now.